installer/uninstaller.sh

```diff
@@ -43,25 +43,45 @@ helm_arr=()
 K8S_NS_ARR=()
 
 cd ../scripts
-./imm -k rm t default-tenant
-cd ../installer
+if [[ "$quiet" != "yes" ]]; then
+    echo "Uninstalling IMM. Following components will be removed:"
+    echo "*Default tenant with it's resources(models,endpoints etc.)"
+    echo "*Helm releases (mgt-api, crd etc.)"
+    echo "Are you sure you want to uninstall IMM? Y/n"
```

Comment:
you could do:
```
read -p   "Are you sure you want to uninstall IMM? Y/n" DELETE_IMM
[[ -z $DELETE_IMM ]] && DELETE_IMM='Y'
```

Reply:
As mentioned, I thought it would be a good idea to require explicit "Y" as a confirmation that user is fully aware of what is going to happen. I know it would be faster to assume "Y" as default value, but I don't think that faster is better in this case.

Reply:
So change it for `echo "Are you sure you want to uninstall IMM? y/N"` as capital letter is suggesting default option